fog/view/server/src/router_request_handler.rs

```diff
@@ -168,7 +171,11 @@ where
 
     let mut query_responses: Vec<MultiViewStoreQueryResponse> = Vec::with_capacity(shards.len());
     let mut remaining_tries = RETRY_COUNT;
+    let _timer = ROUTER_QUERY_REQUESTS.start_timer();
     while remaining_tries > 0 {
+        if remaining_tries < RETRY_COUNT {
+            CLIENT_QUERY_RETRIES.inc();
+        }
```

Comment:
IIUC what this if statement is doing is only increasing the RETRIES counter if this is not the first iteration of the loop. I think it's more readable if its done after the if + break block down below.